consensus/consensus-types/src/block_info.rs

```diff
@@ -39,14 +41,16 @@ impl BlockInfo {
         executed_state_id: HashValue,
         version: Version,
         timestamp_usecs: u64,
+        next_validator_set: Option<ValidatorSet>,
```

Comment:
this contains everything of a `ExecutedState` in `executor/lib.rs`, can we move that simple struct in consensus_types instead?

Reply:
Agreed, this API needs to be sharpened before we wrap up. Mind if we punt on this until we complete the end-to-end with inclusion proof checking? I also think there is at least one PR out there to refactor execution / storage code.